### ISSUE TITLE
Builder crash in a large project

### ISSUE THREAD

**Issue by kellers76** [ImpulseAdventure/GUIslice-Builder#218]
Hi,

I have a big project with currently 48 pages and a lot of variables So far, the builder ran smoothly and is a great thing to generate the code. 
Unfortunately, now suddenly came a builder crash which I do not understand or can fix myself. 
In the log file, the first line says: "class java.lang.ArrayIndexOutOfBoundsException: Index 256 out of bounds for length 256 "

so that I assume that I have reached a limit. Can you help me with that? I would like to create more pages in my project

would be great if you could help me there... 

I can send you further files directly by e-mail if you want

best regards

stefan

**Comment by Pconti31** [comment]
@kellers76 No problem.  I just need you to zip up a few files for me and if necessary I'll patch your project file.
I need your *.prj file, GUIsliceBuilder/logs folder, and the gui_backup folder.
Paul--

**Comment by kellers76** [comment]
Hallo Paul,

Enclosed I send you the files.

I use long names for my variables and fields, unfortunately I am not a professional like you...

I hope you can fix it.

Looking forward to your answer

Best regards


Stefan

⁣BlueMail for Android herunterladen ​

Am 13. Nov. 2022, 18:12, um 18:12, Paul Conti ***@***.***> schrieb:
***@***.*** No problem.  I just need you to zip up a few files for me
>and if necessary I'll patch your project file.
>I need your *.prj file, GUIsliceBuilder/logs folder, and the gui_backup
>folder.
>Paul--
>
>
>--
>Reply to this email directly or view it on GitHub:
>https://github.com/ImpulseAdventure/GUIslice-Builder/issues/218#issuecomment-1312778854
>You are receiving this because you were mentioned.
>
>Message ID:
***@***.***>

**Comment by Pconti31** [comment]
@kellers76 Sorry, but no zip file was attached.  Do not try and post it using email. Go to the web site 
[Issue 218](https://github.com/ImpulseAdventure/GUIslice-Builder/issues/218)
and drag and drop the zip file onto a new post.

**Comment by kellers76** [comment]
Sorry, I thought it goes directly by mail...
[builder_log.zip](https://github.com/ImpulseAdventure/GUIslice-Builder/files/9998028/builder_log.zip)
[Sepsu_200_001_16.zip](https://github.com/ImpulseAdventure/GUIslice-Builder/files/9998032/Sepsu_200_001_16.zip)

**Comment by Pconti31** [comment]
@kellers76 Ok, I found the problem.
```
    listOfCases = new ArrayList[256];
```
I change it to:
```
    listOfCases = new ArrayList[1024];
```
and you will be ok.  I'll do a Release 17b15 with the fix today.
Paul--

**Comment by Pconti31** [comment]
@kellers76 Fixed in Release 0.17.b15
Good Luck!
